### PR TITLE
hardening/1115_create_management_interface_utils_class

### PR DESCRIPTION
**Pull request by pcoello25**
* Fix issue #1115 
* 100% Unit tests passed:
```
Results : Tests run: 66, Failures: 0, Errors: 0, Skipped: 0
``` 
* Assignee: @frbattid

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,2 +1,3 @@
 - [cygnus-ngsi][bug] Change mysql and mongo hosts to "iot-mysql" and "iot-mongo" (#1105)
 - [cygnus-common][hardening] Add implementation for /admin/log route (#807)
+- [cygnus-common][hardening] Create ManagementInterfaceUtils.java (#1115)
```
Comment (frbattid) [inline CHANGES_NEXT_RELEASE]: Create ManagementInterfaceUtils class

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/ManagementInterfaceUtils.java [hunk 2]
```diff
@@ -0,0 +1,226 @@
+/*
```
Comment (frbattid) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/ManagementInterfaceUtils.java]: Add Telefonica's license header!!

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/ManagementInterfaceUtils.java [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+import javax.servlet.http.HttpServletRequest;
+import org.slf4j.MDC;
+
+/**
+ *
+ * @author pcoello25
+ */
+public final class ManagementInterfaceUtils {
+
+/**
```
Comment (frbattid) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/ManagementInterfaceUtils.java]: Wrong indentation

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/ManagementInterfaceUtils.java (hunk 2 above)
Comment (frbattid) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/ManagementInterfaceUtils.java]: This class should be under cygnus-common/src/main/java/com/telefonica/iot/cygnus/management/, it is not a global utility class (that's the purpose of cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/).

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (pcoello25) [inline CHANGES_NEXT_RELEASE]: Fixed in  1b18903

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/ManagementInterfaceUtils.java (hunk 2 above)
Comment (pcoello25) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/ManagementInterfaceUtils.java]: Reubicated :)
Fixed in  c9251eb

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/ManagementInterfaceUtils.java (hunk 3 above)
Comment (pcoello25) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/ManagementInterfaceUtils.java]: Fixed in  c9251eb

**Comment by frbattid** [comment]
LGTM

**Pull request merged**